### ISSUE TITLE
[Bug Report] OpenAll does not work properly with dynamic data

### ISSUE THREAD

**Issue by makeit-mh** [vuetifyjs/vuetify#6341]
Versions and Environment
Vuetify: 1.4.4
Vue: 2.5.16
Browsers: Chrome 71.0.3578.98
OS: Windows 10

Steps to reproduce
Create a dummy node for example "Root".
Add children to this node dynamically.
Nodes are not open even if open-all is set to true.

Expected Behavior
All nodes with children are open

Actual Behavior
Nothing change after add children

Reproduction Link
https://codepen.io/anon/pen/daRoRP?&editable=true&editors=101

Suggestion
I looking for usings 'openAll' property and only one place which can i found is in 'mounted' method of VTreeview.ts. Maybe it should be call in another place for example after rebind items or openAll should be public method. When i call code:
"Object.keys(this.nodes).forEach(key => this.updateOpen(getObjectValueByPath(this.nodes[key].item, this.itemKey), true))"
after load items dynamically it's works!
 Sorry for my English.

**Comment by nekosaur** [comment]
This might or might not be a bug, depending on how we look at the intended function of the prop. I always saw it as `open-all-on-mount` in which case it is working as intended. But I think this is at least the second time this has come up, so perhaps other people expect it to be `open-all-on-load`.

Thoughts @vuetifyjs/core-team ?

**Comment by jacekkarczmarczyk** [comment]
Agree, it's for me more like feature request. I wonder on something like `open-all="initial"` or `open-all="always"` (the second option would disable folding, not sure if this is what @makeit-mh wants to achieve)

**Comment by makeit-mh** [comment]
I think " `open-all="initial"` or `open-all="always"` " it's good idea. I want open all nodes with children after load, not only after mount. I expect that after load data from server vtreeview open all nodes with children which has set 'isOpen' to true. Thats would be great ;-)

**Comment by jacekkarczmarczyk** [comment]
The question is do you want nodes always to be open and disable folding (`open-all="always") or another option - open nodes that are added and still allow folding (`open-all="update"`?)

**Comment by makeit-mh** [comment]
Generally i want to open-all="always", it would be good enough in my opinion ;-). But more precision, only this nodes which has children and 'isOpen' property is set to true, i think it will be more configurable.

**Comment by jacekkarczmarczyk** [comment]
1.5 will be released tomorrow so it might be to late for altering the `open-all` prop behaviour, but i've made a method that opens all nodes public (https://github.com/vuetifyjs/vuetify/pull/6363), so if it's merged you will be able to call `updateAll(true)` after updating items so that they will be all open. Other feature probably will have to wait for next minor release

**Comment by makeit-mh** [comment]
Great ;-) dziękuje ;-)

**Comment by jacekkarczmarczyk** [comment]
Duplicate of https://github.com/vuetifyjs/vuetify/issues/6113
